### PR TITLE
feat(docs): split Okta SCIM docs from SAML page

### PR DESCRIPTION
**Pull request by JoshFerge**
Received a round of feedback from Okta on our docs, some points addressed:
- They want it on a separate page than the SAML setup, so had to move the Okta SSO into its own folder.
- Added concrete "Supported Features", "Requirements", and a "Known Issues/Troubleshooting" section.
- Add a bulletpoint / screenshot on setting the username format to email

I will likely move the azure SCIM docs onto their own page as well but will do so in a separate PR to minimize diff.

### CODE REVIEW

File: src/docs/product/accounts/sso/okta-sso/index.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
```
Comment (JoshFerge) [inline src/docs/product/accounts/sso/okta-sso/index.mdx:1]: this is renamed from `src/docs/product/accounts/sso/okta-sso.mdx` -- the only difference in the file is i've removed the SCIM section at the bottom.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+### Supported Features
+- Create users
```
Comment (PeloWriter) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:15]: Add a space after the heading
```suggestion

- Create users
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- Deactivate users
+- Push groups
+- Import groups
+
+### Requirements
+- You must be on a Sentry Business Plan or above.
+- You must have configured SAML SSO for okta. See [documentation here](/product/accounts/sso/okta-sso/), or the Okta sign-on tab in your Sentry Okta application for configuring SAML.
+- In your Sign On configuration, **You must select Email for the Application username format** on the Sign On application tab in Okta.
```
Comment (PeloWriter) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:23]:
```suggestion
### Requirements

Okta Skim provisioning requires:
- A subscription to Sentry Business Plan or higher.
- Configuration of SAML SSO for Okta[as documented here](/product/accounts/sso/okta-sso/), or use the Okta sign-on tab in your Sentry Okta application to configure SAML.
- Selection of **You must select Email for the Application username format** in the Sign On application tab in Okta.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+As a result of these changes, users who are assigned will be sent an invitation email. When a user is un-assigned, their membership object in Sentry will be deleted.
+
+You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
+
+### Known Issues / Troubleshooting
+- Sentry does not currently support setting any User attributes other than `userName` and `active`
+- The User `name` attribute will contain fields firstName and lastName with the values of `N/A`. Sentry's SCIM API does not currently support these fields but returns them for compatibility purposes.
+- Setting `active` to `false` on a User will delete the organization member record associated with the user.
+- The only filter operation supported for resources is `eq`.
+- When provisioning a new team, Sentry will normalize and convert the team `displayName` uppercase to lowercase, and spaces to dashes.
```
Comment (PeloWriter) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:67]:
```suggestion
- When provisioning a new team, Sentry will both normalize and convert the team `displayName` uppercase to lowercase, and convert spaces to dashes.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- The GET /Groups endpoint will return a maximum of 10000 members in a group.
+- See [SCIM API documentation](/api/scim/) for more information.
```
Comment (PeloWriter) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:69]: If this is related to the GET /Groups endpoint, combine these two bullets.

**Review by PeloWriter — APPROVED** [review]
Thanks, @JoshFerge

**Pull request merged**